Modules/Axis/Sources/Axis/String/UTF8.swift

```diff
@@ -0,0 +1,54 @@
+extension UTF8 {
+    public static var whitespaceAndNewline: Set<UTF8.CodeUnit> = [32, 10, 9, 13]
```

Comment:
I was thinking about this.  Problem with `UnicodeScalars.whitespaceAndNewline.utf8()` is that it will be generating `Set<UTF8.CodeUnit>` every time it's called, which makes little sense as the thing is static anyway.  But I think I found an in-between solution.  Will submit with an update.